packages/auth/src/platform_browser/recaptcha/recaptcha.ts

```diff
@@ -69,3 +70,28 @@ declare global {
     grecaptcha?: Recaptcha | GreCAPTCHATopLevel;
   }
 }
+
+export class RecaptchaConfig {
+  /**
+   * The reCAPTCHA site key.
+   */
+  siteKey: string = '';
+
+  /**
+   * The reCAPTCHA enablement status of the {@link EmailAuthProvider} for the current tenant.
+   */
+  emailPasswordEnabled: boolean = false;
+
+  constructor(response: GetRecaptchaConfigResponse) {
+    if (response.recaptchaKey === undefined) {
+      throw new Error('recaptchaKey undefined');
+    }
+    // Example response.recaptchaKey: "projects/proj123/keys/sitekey123"
+    this.siteKey = response.recaptchaKey.split('/')[3];
+    this.emailPasswordEnabled = response.recaptchaEnforcementState.some(
+      enforcementState =>
+        enforcementState.provider === 'EMAIL_PASSWORD_PROVIDER' &&
+        enforcementState.enforcementState !== 'OFF'
```

Comment:
i think we should separately check for enforcementState to equal audit or enforce otherwise if backend sends an invalid state, this will fail

Reply:
It's a good point. We were actually discussing there might be changes to the behaviour in different modes. I'll make a follow up fix after the preview launch.

Reply:
Great thanks.